packages/ndla-ui/src/Resource/BlockResource.tsx

```diff
@@ -67,10 +67,14 @@ const BlockDescription = styled.p`
   }
 `;
 
-const RightRow = styled(Row)`
+interface TagsAndActionProps {
+  hasMenuButton: boolean;
+}
+
+const TagsAndActionMenu = styled(Row)<TagsAndActionProps>`
   z-index: 1;
   justify-content: flex-end;
-  margin: 0 -${spacing.small} -${spacing.small} 0;
+  margin: 0 -${(props) => (props.hasMenuButton ? spacing.small : 0)} -${spacing.small} 0;
```

Comment:
Hadde en alternativ løsning vært å ha en :last-child / :last-of-type som har padding?

Reply:
Yeah, men tror det får komme i senere runde

Reply:
Dette ble fikset i samme slengen